ui/app/templates/components/oidc/assignment-form.hbs

```diff
@@ -29,6 +29,7 @@
 {{/unless}}
 <form {{on "submit" (perform this.save)}}>
   <div class="box is-sideless is-fullwidth is-marginless">
+    <MessageError @errorMessage={{this.errorBanner}} class="has-top-margin-s" />
```

Comment:
Make sure if there is an errorMessage it conditionally adds the red box around the form field in question. I'm not sure from the screen shot on the PR description if that's the case or if the blue highlight is covering it up.

Reply:
This is surfacing backend API errors - so no red box is added to form fields, but it is added for validation errors!